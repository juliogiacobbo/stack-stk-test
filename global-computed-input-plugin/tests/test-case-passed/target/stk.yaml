applied_templates:
- inputs:
    project_name: testStack
  template_data_path: stacklifecycle-oscli-test-catalog/global-computed-input-template
global_computed_inputs:
  global_computed_input: Teststack
global_inputs: {}
stack: stacklifecycle-oscli-test-catalog
stack_type: app
