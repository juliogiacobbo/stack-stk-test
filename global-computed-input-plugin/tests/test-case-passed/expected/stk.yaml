applied_templates:
- inputs:
    project_name: testStack
  template_data_path: stacklifecycle-oscli-test-catalog/global-computed-input-template
- inputs: {}
  template_data_path: stacklifecycle-oscli-test-catalog/global-computed-input-plugin
global_computed_inputs:
  global_computed_input: Teststack
global_inputs: {}
stack: stacklifecycle-oscli-test-catalog
stack_type: app
